### PR TITLE
PDF error fixing and improvements

### PR DESCRIPTION
**Pull request by andvirga**
This PR contains changes requested for [CHI-515](https://bugs.benetech.org/browse/CHI-515).

Primary reviewer: @GPaoloni 

Changes summary:

1. Added a tiny space between the two text columns that makes long texts occupy another line instead of being too long (for example: Is the perpetrator still in contact with the child? now occupies two lines).
2. Renamed "OK for case worker to call?" to "May social worker call/SMS?" and fixed the incorrect value display (this field is now a multi checkbox and its values can be: Yes, No, Unknown).

### CODE REVIEW

File: plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx [hunk 1]
```diff
@@ -26,10 +26,12 @@ const CasePrintSection: React.FC<Props> = ({ sectionName, values, definitions, u
         {definitions.map((def, i) => {
           return (
             <View key={i} style={i % 2 === 0 ? styles.sectionItemRowOdd : styles.sectionItemRowEven}>
-              <Text style={styles.sectionItemFirstColumn}>{def.label}</Text>
-              <Text style={styles.sectionItemSecondColumn}>
-                {presentValue(unNestInfo ? unNestInformation(def, values) : values[def.name])(def)}
-              </Text>
+              <View style={styles.sectionItemFirstColumn}>
+                <Text style={{ width: '95%' }}>{def.label}</Text>
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx:30]: This does not seem to have the desired effect (at least in my monitor). Maybe try to do some padding/margin right instead?
Screenshot from 2021-03-24 18-28-25

**Review by GPaoloni — APPROVED** [review]
Looks good! Just one small comment.

File: plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx (hunk 1 above)
Comment (andvirga) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx]: Can you please test again this PR with the latest changes that I've pushed. It's weird but reducing the screen size (using Chrome responsive function) is producing that the PDF is not being shown at all (no error traces, and no extra info).
I wonder if this issue is also happening to you.

Comment (GPaoloni) [reply plugin-hrm-form/src/components/case/casePrint/CasePrintSection.tsx]: Seems fixed for me! (resolution at 1366x768)
Screenshot from 2021-03-25 21-39-50
Screenshot from 2021-03-25 21-39-42

**Pull request merged**